base58/base58bench_test.go

```diff
@@ -11,15 +11,21 @@ import (
 	"github.com/btcsuite/btcutil/base58"
 )
 
-func BenchmarkBase58Encode(b *testing.B) {
+var result string
+var resultb []byte
+
+func BenchmarkBase58FastEncode(b *testing.B) {
 	b.StopTimer()
 	data := bytes.Repeat([]byte{0xff}, 5000)
 	b.SetBytes(int64(len(data)))
 	b.StartTimer()
 
+	var s string
 	for i := 0; i < b.N; i++ {
-		base58.Encode(data)
+		s = base58.Encode(data)
```

Comment:
Please revert this change.  It is not necessary and does not change the results at all.

With:
```
$ go test -bench=BenchmarkBase58 -benchmem
BenchmarkBase58FastEncode-6           20          71934135 ns/op           0.07 MB/s       20736 B/op          3 allocs/op
```

Without:
```
$ go test -bench=BenchmarkBase58 -benchmem
BenchmarkBase58FastEncode-6           20          72284180 ns/op           0.07 MB/s       20736 B/op          3 allocs/op
```